examples/features/stats_monitoring/client/main.go

```diff
@@ -0,0 +1,97 @@
+// Binary client is an example client.
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/stats"
+
+	pb "google.golang.org/grpc/examples/features/proto/echo"
+)
+
+var addr = flag.String("addr", "localhost:50051", "the address to connect to")
+
+// *statsHandler implements [stats.Handler](https://pkg.go.dev/google.golang.org/grpc/stats#Handler) interface
+type statsHandler struct{}
+
+// TagRPC can attach some information to the given context.
+// The context used for the rest lifetime of the RPC will be derived from the returned context.
+func (st *statsHandler) TagRPC(ctx context.Context, stat *stats.RPCTagInfo) context.Context {
+	log.Printf("[TagRPC] RPC request send to: %s", stat.FullMethodName)
```

Comment:
How about adding something to the context here, and showing that the same value is present in the context passed to `HandleRPC`?

Also, could you please print the complete `stats.RPCTagInfo` instead of only `stat.FullMethodName`.